### PR TITLE
Fix Job scheduler which keeps adding new jobs

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
The code was comparing ms and second.
Causes the job scheduler to always add new jobs. 

## How
Fix and make it foolproof
Fix the UI

### CODE REVIEW

**Review by cgardens — APPROVED** [review]
this works for me! would it make life easier though if we just used millis everywhere instead?

File: dataline-webapp/src/pages/SourcesPage/pages/AllSourcesPage/components/LastSyncCell.tsx [hunk 1]
```diff
@@ -16,20 +16,20 @@ const Content = styled.div<{ enabled?: boolean }>`
 `;
 
 type IProps = {
-  value: number;
```
Comment (Jamakase) [inline dataline-webapp/src/pages/SourcesPage/pages/AllSourcesPage/components/LastSyncCell.tsx:19]: What's the reason behind this change? Don't think this renaming is required.

Comment (Jamakase) [reply dataline-webapp/src/pages/SourcesPage/pages/AllSourcesPage/components/LastSyncCell.tsx:19]: In case we want to bring more meaning to this value field for Cell components - I think then we should get rid of value part and use just `seconds` + similar changes should be applied in other cells too.

**Review by jamakase — APPROVED** [review]

File: dataline-webapp/src/pages/SourcesPage/pages/AllSourcesPage/components/LastSyncCell.tsx (hunk 1 above)
Comment (michel-tricot) [reply dataline-webapp/src/pages/SourcesPage/pages/AllSourcesPage/components/LastSyncCell.tsx:19]: might not be necessary for all the other cells but when we manipulate time as long or integer, we need to have some sort of signal that it is second or millisecond.
I'll rename this one to `timeInSecond`
